### ISSUE TITLE
[Bug]: Can't click on Outlook service, no reaction

### ISSUE THREAD

**Issue by Malkierian** [getferdi/ferdi#2287]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/getferdi/ferdi/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/getferdi/ferdi/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/getferdi/ferdi/issues) for a feature request that matches the one I want to file, without success.

### Ferdi Version

5.6.3

### What operating system are you using?

Windows

### Server

Using without an account

### Operating System Version

Windows 10 pro 21H2

### What arch are you using?

x64

### Git SHA and/or build date

yab8abe

### Steps to reproduce

Fresh install.
Go to Add New Service.
Click on Outlook (not Office 365)
...

### Expected Behavior

Allow me to add Outlook accounts

### Actual Behavior

Absolutely nothing after the search

**Comment by vraravam** [comment]
@Malkierian - we do not have / support any "outlook" service/recipe. The only one that Ferdi supports is called "Office 365". If its a custom recipe, unfortunately, we will not be able to debug it.

**Comment by Malkierian** [comment]
It was literally a fresh install, I didn't even click on the Custom Services tab.
![image](https://user-images.githubusercontent.com/774771/143804262-caa7cde2-1ee4-465d-bc39-f1a28cbdfa48.png)

**Comment by vraravam** [comment]
ok - that's a known issue.

We had merged `Outlook` and `Office365` into a single recipe a few months ago.

The way Ferdi works is that it tries to pull in recipes from the server as well as what's packaged in the installer. The server is quite outdated unfortunately, and we are waiting for @vantezzen or @kytwb to deploy the latest (afaik this hasn't been done recently).
Closing this as a known bug for now.
